docs/src/pages/guides/FHIRTerminologyGuide.md

```diff
@@ -167,3 +163,7 @@ Example configuration:
                 }]
             }]
 ```
+
+## FHIR Terminology Service Diagram
```

Comment:
my only comment is that maybe this should be higher / more prominent.  its a great diagram.

Reply:
Moved the diagram to the top of the guide after the opening paragraph.